components/AddEventForm/index.js

```diff
@@ -2,53 +2,54 @@ import { useState } from "react";
 import styled from "styled-components";
 
 export default function AddEventForm({ onSubmit, appointment }) {
+  const [selectedType, setSelectedType] = useState();
+
   const StyledForm = styled.form`
+    width: 80%;
+    margin: 1rem auto 1rem 3.4rem;
     display: flex;
     flex-direction: column;
-    border: 2px solid black;
-    width: 95%;
-    font-size: 1.2rem;
-    align-items: center;
+    background-color: hsla(0, 0%, 4%, 0.64);
+    color: lightgray;
     border-radius: 2rem;
-    background-color: #ccd9ff;
-    margin: 1rem auto;
-    padding 1rem 1rem;
   `;
+  const StyledSubmitButton = styled.button`
+    width: 3rem;
+  `;
+  const StyledParagraph = styled.p`
+    align-self: center;
+    font-size: 1.4rem;
+  `;
+  const StyledLabel = styled.label`
+    // color: red;
+    // border: 3px solid red;
+  `;
+  function handleChangeType(event) {
+    event.preventDefault();
+    setSelectedType(event.target.value);
+    return true;
+  }
 
   return (
     <StyledForm onSubmit={onSubmit}>
-      <p>Please fill in the form:</p>
-      {appointment ? (
-        <label>
-          Type:
-          <select
-            name="type"
-            onChange={(event) => {
-              handleChangeType(event);
-            }}
-            defaultValue={appointment.type}
-          >
-            <option value="wishlist">Wishlist</option>
-            <option value="latest">Latest</option>
-            <option value="upcoming">Upcoming</option>
-          </select>
-        </label>
-      ) : (
-        <label>
-          Type:
-          <select
-            name="type"
-            onChange={(event) => {
-              handleChangeType(event);
-            }}
-            defaultValue={"wishlist"}
-          >
-            <option value="wishlist">Wishlist</option>
-            <option value="latest">Latest</option>
-            <option value="upcoming">Upcoming</option>
-          </select>
-        </label>
-      )}
+      <StyledParagraph>Please fill in the form:</StyledParagraph>
+
+      <StyledLabel>
```

Comment:
since there is no styling in this styled component, it can be a normal label

```suggestion
      <label>
```